notebooks/04.02-Simple-Scatter-Plots.ipynb

```diff
@@ -37,10 +37,8 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">a = b + C
</pre><p>Is this the addition you want?</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>